tutorials/Window.getNativeId.md

```diff
@@ -0,0 +1,17 @@
+Returns then running applications uuid
```

Comment:
This returns the native OS level Id, in Windows it will return the Windows hwnd.